src/Components/ErrorPopUp/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import styles from './style.module.css';
+
+function ErrorPopUp(props) {
+  const [show, setShow] = useState(true);
+  const [errorMessage, setErrorMessage] = useState('Something went wrong while processing the application status update. Please try again.');
+  const disapear = () => setShow(false);
```

Comment:
not a huge deal but should be spelled 'disappear'

Reply:
however the `setShow` hook that you created should be called directly from the `onClick` method on line 28, no need to wrap it in a function

Reply:
Oyy... Wish I could make those mistakes disappear : )

Reply:
haha no youre good